libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405WING-bdshot/hwdef-bl.dat

```diff
@@ -0,0 +1 @@
+include ../SpeedyBeeF405WING/hwdef-bl.dat
```

Comment:
Remove this file in its entirety, please - not needed when you're saying `USE_BOOTLOADER_FROM_BOARD` in the main hwdef.

Please note you can clean the commit list up and force-push your existing branch; see https://ardupilot.org/dev/docs/git-interactive-rebase.html